Sloth.Web/Views/Shared/_TransactionsTable.cshtml

```diff
@@ -13,20 +13,20 @@
 <table id="transactionsTable" class="table table-bordered">
     <thead>
         <tr>
-            <th>@Html.DisplayNameFor(x => x.Id)</th>
-            <th>@Html.DisplayNameFor(x => x.TransactionDate)</th>
-            <th>@Html.DisplayNameFor(x => x.KfsTrackingNumber)</th>
-            <th>@Html.DisplayNameFor(x => x.DocumentNumber)</th>
-            <th>@Html.DisplayNameFor(x => x.ProcessorTrackingNumber)</th>
-            <th>@Html.DisplayNameFor(x => x.MerchantTrackingNumber)</th>
-            <th>@Html.DisplayNameFor(x => x.Description)</th>
+            <th>@Html.DisplayNameFor(x => x.Transactions.FirstOrDefault().Id)</th>
+            <th>@Html.DisplayNameFor(x => x.Transactions.FirstOrDefault().TransactionDate)</th>
+            <th>@Html.DisplayNameFor(x => x.Transactions.FirstOrDefault().KfsTrackingNumber)</th>
+            <th>@Html.DisplayNameFor(x => x.Transactions.FirstOrDefault().DocumentNumber)</th>
+            <th>@Html.DisplayNameFor(x => x.Transactions.FirstOrDefault().ProcessorTrackingNumber)</th>
+            <th>@Html.DisplayNameFor(x => x.Transactions.FirstOrDefault().MerchantTrackingNumber)</th>
+            <th>@Html.DisplayNameFor(x => x.Transactions.FirstOrDefault().Description)</th>
```

Comment:
I have a bad feeling this might do a whole bunch of queries, since Transactions is a list.  Take a look at the logs and if it isn't firing queries, then ok.  Otherwise you might have to just use a local variable to create a dummy transaction so you can do the displayfor

Reply:
I believe the Linq expression in DisplayNameFor() is used to just get the string representation of the referenced member, but I'll check to make sure...

Reply:
In the past, I've got the first one 

> Also, I meant to ask if a transaction status should have any bearing on whether the webhook is available to called?

No, at least not for the Registration hook

Reply:
Yep, that's what it does, along with use of a helper to html-encode the strings.